rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -16,8 +19,8 @@ void opcode_0000(const ScriptArguments& args) {
 */
 void opcode_0001(const ScriptArguments& args, const ScriptInt time) {
 	RW_UNIMPLEMENTED_OPCODE(0x0001);
-	RW_UNUSED(time);
-	RW_UNUSED(args);
+	auto thread = args.getThread();
+	thread->wakeCounter = time > 0 ? time : -1;
```

Comment:
Beyond the scope of PR: Did we verify the behaviour with negative wait? Shall we assert instead?